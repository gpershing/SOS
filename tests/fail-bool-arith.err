Fatal error: exception Failure("Cannot add or subtract bool and bool")
