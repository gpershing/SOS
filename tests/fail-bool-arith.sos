t: bool = true
f: bool = false

sum: bool = t + f
