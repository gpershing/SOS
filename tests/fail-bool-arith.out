Cannot add or subtract bool and bool
